seed/seed.go

```diff
@@ -157,6 +157,9 @@ type Seed interface {
 	// Iter provides a way to iterately perform a function on
 	// each of the snaps for which LoadMeta loaded their metadata.
 	Iter(f func(sn *Snap) error) error
+
+	// Auto import assertion from root of seed if grade is dangerous
+	LoadAutoImportAssertion(commitTo func(*asserts.Batch) error) error
```

Comment:
I forgot, as this really potentially imports many assertions it should be called LoadAutoImportAssertions plural, it also less confusing vs the other method of the same name that way.

Reply:
Function name changes to reflect the fact many assertions can be imported.